apps/i18n/common/en_us.json

```diff
@@ -1419,7 +1428,8 @@
   "plLandingDescription": "Thank you for completing a {course} workshop. Please complete this survey about your experience to help us improve future professional development workshops!",
   "plLandingHeading": "My Professional Learning",
   "plLandingSubheading": "Submit your feedback",
-  "plLandingStartSurvey": "Start survey",
```

Comment:
did you mean to remove this one? it looks like its still being used on the landing page

Reply:
Nope. That was a mistake. Probably happened when I was searching for the strings to add. I will fix it